src/main/webapp/modal-overlay.css

```diff
@@ -45,5 +45,5 @@
   left: 0;
   right: 0;
   bottom: 0;
-  background-color: black;
+  background-color: rgba(0,0,0,.5);
```

Comment:
worth commenting the approximate color on these so that we can make sense of them.

Reply:
Addressed: d7665f2